src/events/guildMemberRemove.js

```diff
@@ -29,6 +29,7 @@ module.exports = {
             if(memberCount <= config.membersForRole){
                 await client.guilds.cache.get(config.guild).roles.delete(memberDoc.guild.role);
                 await guildModel.findByIdAndUpdate(memberDoc.guild._id, {$set: {role: null}});
+                client.emit('updateGuilds')
```

Comment:
explicacao no primeiro comentario em src/events/updateGuilds.js
```suggestion
                client.emit('updateGuilds', false);
```